app/src/main/java/org/simple/clinic/crash/SentryCrashReporter.kt

```diff
@@ -53,6 +57,15 @@ class SentryCrashReporter @Inject constructor(
             { report(it) })
   }
 
+  @SuppressLint("CheckResult")
+  private fun identifyCurrentCountryCode() {
+    val countryCode = appConfigRepository
+        .currentCountry()
+        .get().isoCountryCode
+
+    Sentry.getContext().addTag("countryCode", countryCode)
```

Comment:
Can you tell me how do we check in Sentry if the country code is identified correctly?

Reply:
I tested it [here](https://sentry.io/organizations/resolve-to-save-lives/issues/2200486205/?environment=sandbox&project=1212614&query=is%3Aunresolved) in a test branch